core/components/molecules/resource-list/item/item.js

```diff
@@ -189,6 +190,8 @@ ListItem.propTypes = {
   image: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
   /** An icon to display as a thumbnail image for the list item. */
   icon: PropTypes.oneOf(__ICONNAMES__),
+  /** Defines the flex-basis of the middle column. Accepts any number and unit, for example: `40%` or `200px` */
+  bodyWidth: PropTypes.string,
```

Comment:
I think we could change this for a function that expects a `positive integer number` + `%` or `px`. So we tighten a little bit more the openness of the string :)